scripts/zones/Sealions_Den/mobs/Ultima.lua

```diff
@@ -15,8 +15,14 @@ entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.GIL_MAX, -1)
 end
 
+entity.onMobSpawn = function(mob)
+    mob:setLocalVar("stage", 1)
```

Comment:
if we call the 1st stage `0` instead of `1` you can omit setting it on spawn (things not set yet equal 0 when checked)

Reply:
I think enough programmers and even amateurs are familiar enough with starting at 0